src/main/java/com/tileman/TilemanModeMinimapOverlay.java

```diff
@@ -60,7 +60,7 @@ private TilemanModeMinimapOverlay(Client client, TilemanModeConfig config, Tilem
 	@Override
 	public Dimension render(Graphics2D graphics)
 	{
-		if (!config.drawTileOnMinimmap())
+		if (!config.drawTilesOnMinimmap())
```

Comment:
change from minimap MR. Maybe remove, or wait until the other MR is merged

Reply:
Changed in other MR